host-spec/ae-hostapi.tm

```diff
@@ -1686,7 +1680,8 @@
   <strong|Arguments>:
 
   <\itemize>
-    <item><verbatim|result>: a pointer to the buffer containing the 256-bit
+    <item><verbatim|result>: a regular <todo|What is a regular pointer? is it
+    a well know wasm concept>pointer to the buffer containing the 256-bit
```

Comment:
I think by a regular pointer we just mean a pointer of the used architecture, which in wasm32 is just a 32-bit pointer to a memory address. Maybe just replace "regular" with "32-bit". Or add a pointer definition before the pointer-size definition and link to it.